tests.py

```diff
@@ -0,0 +1,13 @@
+from tvl2018 import *
```

Comment:
Google style says don't do * import.. .as it is hard to tell which things are actually part of the library.  So so import * as tvl, and then use tvl. everywhere.  OK?